<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>0</y>
      <width>260</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Square">
     <attribute name="title">
      <string>Square</string>
     </attribute>
     <widget class="QWidget" name="Form" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>510</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="Name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dimensions:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="x">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>32</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="y">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>32</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="length">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>69</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="Length">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Length:</string>
       </property>
      </widget>
      <widget class="QLabel" name="X">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>35</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
      <widget class="QLabel" name="Y">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>35</y>
         <width>10</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PenColor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pen Color:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>109</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>light gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tranparent</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="PenWidth">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pen Width:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PenStyle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pen Style:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PenCapStyle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pen Cap Style:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PenJoinStyle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pen Join Style:</string>
       </property>
      </widget>
      <widget class="QLabel" name="BrushColor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Brush Color:</string>
       </property>
      </widget>
      <widget class="QLabel" name="BrushStyle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Brush Style:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>189</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>no pen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>solid line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dash line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dot line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dash dot line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dash dot dot line</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>269</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>miter join</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bevel join</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>round join</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>309</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>light gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dark yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tranparent</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>349</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>no brush</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>solid pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dense 1 pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dense 2 pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dense 3 pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dense 4 pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dense 5 pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dense 6 pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dense 7 pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hor pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ver pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cross pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bdiag pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fdiag pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>diag cross pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>linear gradient pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>conical gradient pattern</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>radia gradient pattern</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>229</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>flat cap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>square cap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>round cap</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="length_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>149</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Rectangle">
     <attribute name="title">
      <string>Rectangle</string>
     </attribute>
     <widget class="QWidget" name="Form_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>431</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <widget class="QLabel" name="Name_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dimensions:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="x1_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="TopLeft_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Top Left:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="y1_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="y2_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>60</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="x2_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="BottomRight_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Bottom Right:</string>
       </property>
      </widget>
      <widget class="QLabel" name="X_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
      <widget class="QLabel" name="Y_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="CreateShape">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>260</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Shape</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
